scripts/testing/scan_for.sh

```diff
@@ -15,8 +15,16 @@ outfile=timed_scan_$(date  +"%d.%m.%y-%H.%M").csv.gz
 
 echo "
 $(tput setaf 2)
-        Scanning for $duration, output file is $outfile .
+        Scanning for $duration, expect a $outfile.png in $(pwd) afterwards.
 $(tput sgr0)"
 
 $scriptpath/start_scanning.sh $outfile
-nohup bash -c "sleep $duration; $scriptpath/stop_and_finalize_scanning.sh $outfile" 2> /dev/null &
+if ! pidof rtl_power >/dev/null ; then
```

Comment:
You might want to check for both `rtl_power` and `rtl_fm` given the latter is what is actually run by the capture scripts?

Reply:
This is a check if rtl_power succeeded to grab a rtl-sdr.
So it does not matter if the sdr is currently taken by any of the rtl_* commands

Reply:
Oh right, nice - sorry about that, LGTM.